roles/maas_region_controller/tasks/install_maas.yaml

```diff
@@ -100,7 +100,7 @@
   delegate_to: "{{ item }}"
   delegate_facts: true
   loop: "{{ groups['maas_rack_controller'] }}"
-  when: ('maas_rack_controller' in group_names)
+  when: ('maas_rack_controller' not in group_names)
```

Comment:
if `maas_rack_controller` is not in group names on this task, `groups['maas_rack_controller']` will raise a key not in dict exception.

Reply:
Since the readme specifies that the maas_rack_controller is required at least once, and not optional, this might not be an issue. However perhaps it's preferable to re-write this logic completely, what do you think?

Reply:
Ah, i forgot that this was a requirement: I tested this with only a postgres primary and a region role in the hosts file when I had the error.

It might be useful to define a task immediately that runs on all hosts that fails if required groups are not found?

Reply:
I can look into that, yes